WolframLanguage.sublime-syntax

```diff
@@ -47,16 +47,33 @@ variables:
       Catalan|Degree|E|EulerGamma|Glaisher|GoldenAngle|GoldenRatio|I|Khinchin|MachinePrecision|Pi
     )\b
 
+  first_param_functional_functions: |-
+    (?x)
+    Apply|ArrayFilter|
+    BlockMap|
+    FixedPoint|FixedPointList|Fold|FoldList|FoldPair|FoldPairList|
+    Map|MapAt|MapIndexed|MapThread|
+    Nest|NestList|NestGraph|NestWhile|NestWhileList|
+    Scan|SequenceFold|SequenceFoldList
+
+  last_param_functional_functions: |-
+    (?x)
+    CountDistinctBy|LengthWhile|TakeWhile
+
+  parametic_functions: Table|Do|Sum|Product
+
+  scoping_functions: Block|Module|With|Function
+
   nb_code_styles: (Input)
 
 
 contexts:
 
   main:
-    - include: declarations
     - include: expressions
 
   expressions:
+    - include: declarations
```

Comment:
I am no sure we should index all the localized functions.

Reply:
What does "index" mean?
From my perspective, "declarations" have no grammatical difference with other expressions and should not be ruled out.